libr/util/print.c

```diff
@@ -12,6 +12,76 @@
 
 static const char hex[16] = "0123456789ABCDEF";
 
+typedef struct { unsigned char ch; unsigned char hx } chart;
+
+chart gameboy_encoding_chart[60] = {
```

Comment:
this is wrong, there is no such thing as "the gameboy encoding/charset". Games bring their own set, some of them are quite unique. Please do not hard code this here